x/ethbridge/client/cli/tx.go

```diff
@@ -52,10 +50,12 @@ func GetCmdCreateEthBridgeClaim(cdc *codec.Codec) *cobra.Command {
 
 			ethBridgeClaim := types.NewEthBridgeClaim(nonce, ethereumSender, cosmosReceiver, validator, amount)
 			msg := types.NewMsgCreateEthBridgeClaim(ethBridgeClaim)
+			err = msg.ValidateBasic()
```

Comment:
we don't need it as it's already checked on the ante handler

Reply:
fixed in latest push